tools/react-app-template/src/App.tsx

```diff
@@ -15,14 +15,7 @@ export const App: React.FC = () => {
                     <h1>Hello GoodWorld!</h1>
                     <p>
                         Edit <code>/src/App.tsx</code> to get started. Learn more about this template in{" "}
-                        <a
-                            target="_blank"
-                            rel="noreferrer"
-                            href="https://github.com/gooddata/gooddata-ui-sdk/blob/master/tools/react-app-template/README.md"
-                        >
-                            Readme
-                        </a>
-                        .
+                        <code>README.md</code>.
```

Comment:
Should we rather remove it?

Reply:
I think it still adds value - we want to make sure the user would not just skip README altogether.